docs/tutorials/SignalsPrimer.m

```diff
@@ -185,16 +203,16 @@
 b = y + 2;
 
 % Looking at the name of your Signals may help you here
-% TODO use visualizations (sig.Visulizer)
+% TODO use visualizations (sig.Visualizer)
 y.Name
 a.Name
 b.Name
 
 %% Signals can derive from multiple signals
-% The above examples show how a signal can derive from one other signal,
-% however a signal can be defined by any number of other signals as well as
-% by constants.
-% Mathematically, Signals can be viewed as variables which any time they
+% The above examples show how a signal can derive from one other signal;
+% however, a signal can be defined by any number of other signals, as well
+% as by constants.
+% Mathematically, Signals can be viewed as variables which, any time they
 % take a new value, cause any equations dependent equations to be
```

Comment:
Whoops spotted a typo

Reply:
I like punctuation = D
I do think it helps with readability

Reply:
I think all those commas are unnecessary but my type comment would about the following line (which I wrote):

> take a new value, cause any equations dependent equations to be

This should read

> take a new value, cause any dependent equations to be

Reply:
ah ok. Gramatically, I think the punctuation is required : )

Reply:
There are no hard and fast rules for the use of the comma and the meaning does not change one iota with or without those commas but I don't really care: the comment was purely about 'equations' being written twice.

Reply:
In that second sentence, the comma is not necessary; however, I think it certainly helps with readability. 

In the first sentence, the semicolon (or a period) is required, and if not applied could certainly change the meaning of "however".